README.rst

```diff
@@ -133,66 +143,84 @@ For other build systems
+xCORE.ai explorer board and x86 platforms. All unit tests are currently in the `/test/
+<https://github.com/xmos/lib_xcore_math/tree/master/test/>`_ directory:
+
+* `/test/ <https://github.com/xmos/lib_xcore_math/tree/master/test/>`_ - Unit test projects for ``lib_xcore_math``:
+
+  * `bfp_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/bfp_tests/>`_ - BFP unit tests
+  * `dct_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/dct_tests/>`_ - DCT unit tests
+  * `filter_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/filter_tests/>`_ - filtering unit tests
+  * `fft_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/fft_tests/>`_ - FFT unit tests
+  * `scalar_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/scalar_tests/>`_ - Scalar op unit tests
+  * `vect_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/vect_tests/>`_ - Vector op unit tests
+  * `xs3_tests/ <https://github.com/xmos/lib_xcore_math/tree/master/test/xs3_tests/>`_ - XS3-specific unit tests
 
-This project uses CMake to build the unit test applications. Use the steps described above to
-configure and build the unit test applications. Both unit test projects currently target the
-xCORE.ai explorer board. All unit tests are currently in the `/test/
-<https://github.com/xmos/lib_xcore_math/tree/develop/test/>`_ directory:
+All examples are currently in the `/example/
+<https://github.com/xmos/lib_xcore_math/tree/master/example/>`_ directory:
 
-* `/test/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/>`_ - Unit test projects for ``lib_xcore_math``.
+* `/example/ <https://github.com/xmos/lib_xcore_math/tree/master/example/>`_ - Example projects for ``lib_xcore_math``:
 
-  * `bfp_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/bfp_tests/>`_ - BFP unit tests 
-  * `vect_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/vect_tests/>`_ - Vector op unit tests
-  * `scalar_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/scalar_tests/>`_ - Scalar op unit tests
-  * `filter_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/filter_tests/>`_ - filtering unit tests
-  * `fft_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/fft_tests/>`_ - FFT unit tests
-  * `dct_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/dct_tests/>`_ - DCT unit tests
-  * `xs3_tests/ <https://github.com/xmos/lib_xcore_math/tree/develop/test/xs3_tests/>`_ - XS3-specific unit tests
+  * `bfp_demo/ <https://github.com/xmos/lib_xcore_math/tree/master/test/bfp_demo/>`_ - BFP demo
+  * `filter_demo/ <https://github.com/xmos/lib_xcore_math/tree/master/test/filter_demo/>`_ - Filter demo
+  * `fft_demo/ <https://github.com/xmos/lib_xcore_math/tree/master/test/fft_demo/>`_ - FFT demo
+  * `vect_demo/ <https://github.com/xmos/lib_xcore_math/tree/master/test/vect_demo/>`_ - Vect demo
```

Comment:
Please change `Vect demo` to `Vector demo`

Reply:
I renamed to Vector op demo, to match the same working of the unit tests.